README.md

```diff
@@ -109,12 +117,11 @@ To upload a pre-existing database dump into the container with [`nixos-container
 
      Store the value in `.credentials/GH_SECRET`
 
-You only need real GitHub credentials to use the OAuth login feature.
-To get going quickly, set any values for secrets required by the server:
+You currently need the credentials for starting the backend, as it tries to list team memberships on startup.
+To still skip this step, set any values for secrets required by the server:
```

Comment:
Won't that fail? https://github.com/Nix-Security-WG/nix-security-tracker/issues/239

Tested. This will fail without credentials, and this is exactly why you need an access token with members read access.

Reply:
Yeah so I guess it would be better to delete the section and make token generation mandatory?

Reply:
Yes